lib/xcribe.ex

```diff
@@ -79,12 +78,22 @@ defmodule Xcribe do
     generator. Default is `XCRIBE_ENV`.
     * `:json_library` - The library to be used for json decode/encode (Jason
     and Poison are supported). The default is the same as `Phoenix` configuration.
+    * `:serve` - Enable Xcribe serve mode. Default `false`. See more `Serving doc`
   """
   use Application
 
+  alias Xcribe.CLI.Output
+  alias Xcribe.Config
+
   @doc false
   def start(_type, _opts) do
     opts = [strategy: :one_for_one, name: Xcribe.Supervisor]
+
+    case Config.check_configurations([:serve]) do
```

Comment:
We could consider in the future to use https://hexdocs.pm/nimble_options/NimbleOptions.html to validate those options.